src/librawspeed/decoders/IiqDecoder.cpp

```diff
@@ -242,6 +257,47 @@ void IiqDecoder::DecodePhaseOneC(const std::vector<IiqStrip>& strips,
     DecodeStrip(strip, width, height);
 }
 
+void IiqDecoder::CorrectPhaseOneC(ByteStream meta_data, uint32 split_row,
+                                  uint32 split_col) {
+  ByteStream meta_data_base(meta_data.getSubStream(0));
+
+  meta_data.skipBytes(8);
+  const uint32 bytes_to_entries = meta_data.getU32();
+  meta_data.skipBytes(bytes_to_entries - 12);
+  const uint32 entries_count = meta_data.getU32();
+  meta_data.skipBytes(4);
+
+  // this is how much is to be read for all the entries
+  ByteStream entries(meta_data.getStream(16 * entries_count));
+
+  bool quadrant_multiplier_applied = false;
+
+  for (uint32 entry = 0; entry < entries_count; entry++) {
+    const uint32 tag = entries.getU32();
+    const uint32 len = entries.getU32();
+    const uint32 data = entries.getU32();
```

Comment:
`s/data/offset/`

Reply:
Also, either https://github.com/darktable-org/rawspeed/pull/117/files#r153709175, or this is missing `entries.skipBytes(4);`

Reply:
Yeah, I goofed the size calculation